doc/CustomFonts.md

```diff
@@ -0,0 +1,86 @@
+# Registering Custom Fonts
+
+This guide shows how to set up fonts in your Flutter project to work with the Appcues Flutter plugin.
+
+The Appcues SDK loads fonts from the native app bundle using system APIs. This means the fonts must be configured in each of the Android and iOS projects for your app.
+
+- [Registering Custom Fonts](#registering-custom-fonts)
+  - [Naming Font Files](#naming-font-files)
+  - [Android](#android)
+    - [Maintaining a Single Source of Truth for Font Files](#maintaining-a-single-source-of-truth-for-font-files)
+  - [iOS](#ios)
+  - [Summary](#summary)
+
+## Naming Font Files
+
+The file name for a font must match the PostScript name of the font for your flows to properly load the custom font on both Android and iOS.
+This is because in native Android apps, fonts are referenced by their resource name which is the filename[^1], while in native iOS apps, fonts are referenced by their PostScript name[^2].
+
+On macOS, you can find the PostScript name of a font by opening it with the Font Book app and selecting the Font Info tab.
+
+## Android
+
+For an Android app, the font files need to be copied to the expected directory: `android/app/src/main/assets/fonts`.
```

Comment:
not too important but we also accept as valid path coming from android/app/src/main/rest/font

its also possible to provide a custom package list in case they have multi-module (but probably not used for flutter apps)

Reply:
I actually tried `android/app/src/main/res/font` first, but it has requirements about the format of the file name, and we need the file name to match the postscript name exactly for the font to work on both platforms. Unless fonts in `res/fonts` have different rules about how the name gets applied.